index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+  <meta http-equiv="Pragma" content="no-cache" />
+  <meta http-equiv="Expires" content="0" />
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <script lang="js">
+      window.okapi = {
+        url: 'https://folio-etesting-snapshot-kong.ci.folio.org',
+        authnUrl: 'https://folio-etesting-snapshot-keycloak.ci.folio.org',
+      };
+      window.config = {
+        tenantOptions: {
+          diku: {
+            name: 'diku', clientId: 'diku-application'
+          },
+        },
+        preserveConsole: true,
+      };
+    </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/localforage/1.10.0/localforage.min.js"></script>
```

Comment:
Monolithic bundles are totally self-contained so using a CDN is a definite change.